presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml

```diff
@@ -4,3 +4,8 @@ databases:
   presto:
     host: presto-master
     configured_hdfs_user: hive
+    cli_ldap_truststore_path: /etc/ldap/cacerts.jks
```

Comment:
Here's what I found on the topic: http://stackoverflow.com/a/25632724/52142. The only issue would be having perl with that package installed on each dev's machine. That could be done by publishing a busybox/alpine docker image with perl and that module preinstalled, and maybe a wrapper script. So usage would be sth like:

`docker run teradatalabs/yaml-merge file1.yml file2.yml > merged.yml`

Alternatively we could implement yaml merging in tempto and allow passing multiple config files.